--- libfcgi.mak	2002-02-25 02:47:17.000000000 +0100
+++ libfcgi.mak.new	2022-04-20 20:03:40.989524300 +0200
@@ -90,7 +90,7 @@
 BSC32_SBRS= \
 	
 LINK32=link.exe
-LINK32_FLAGS=Ws2_32.lib /nologo /dll /pdb:none /machine:I386 /out:"$(OUTDIR)\libfcgi.dll" /implib:"$(OUTDIR)\libfcgi.lib" 
+LINK32_FLAGS=Ws2_32.lib /nologo /dll /pdb:none /out:"$(OUTDIR)\libfcgi.dll" /implib:"$(OUTDIR)\libfcgi.lib" 
 LINK32_OBJS= \
 	"$(INTDIR)\fcgi_stdio.obj" \
 	"$(INTDIR)\fcgiapp.obj" \
@@ -183,7 +183,7 @@
 <<
 
 LINK32=link.exe
-LINK32_FLAGS=Ws2_32.lib /nologo /dll /profile /map:"$(INTDIR)\libfcgi.map" /debug /machine:I386 /out:"$(OUTDIR)\libfcgi.dll" /implib:"$(OUTDIR)\libfcgi.lib" 
+LINK32_FLAGS=Ws2_32.lib /nologo /dll /profile /map:"$(INTDIR)\libfcgi.map" /debug /out:"$(OUTDIR)\libfcgi.dll" /implib:"$(OUTDIR)\libfcgi.lib" 
 LINK32_OBJS= \
 	"$(INTDIR)\fcgi_stdio.obj" \
 	"$(INTDIR)\fcgiapp.obj" \
